src/core/Akka.Streams/Implementation/IO/OutputStreamSourceStage.cs

```diff
@@ -231,7 +283,14 @@ private void SendData(ByteString data)
         {
             Send(() =>
             {
-                _dataQueue.Add(data);
+                try
+                {
+                    _dataQueue.Add(data);
+                }
+                catch (Exception ex)
```

Comment:
I wanted to stay as close as possible to the jvm  ;-) 
https://github.com/akka/akka/blob/master/akka-stream/src/main/scala/akka/stream/impl/io/OutputStreamSourceStage.scala#L147

Should I report this back?